cortex-js/README.md

```diff
@@ -11,91 +11,39 @@
 > ⚠️ **Cortex is currently in Development**: Expect breaking changes and bugs!
 
 ## About
-Cortex is an openAI-compatible local AI server that developers can use to build LLM apps. It is packaged with a Docker-inspired command-line interface and a Typescript client library. It can be used as a standalone server, or imported as a library. 
+Cortex is an OpenAI-compatible AI engine that developers can use to build LLM apps. It is packaged with a Docker-inspired command-line interface and client libraries. It can be used as a standalone server or imported as a library. 
 
-Cortex currently supports two inference engines:
+Cortex currently supports 3 inference engines:
 
 - Llama.cpp
+- ONNX Runtime
 - TensorRT-LLM
 
-> Read more about Cortex at https://jan.ai/cortex
-
 ## Quicklinks
-Cortex
-- [Website](https://jan.ai/)
-- [GitHub](https://github.com/janhq/cortex)
-- [User Guides](https://jan.ai/cortex)
-- [API reference](https://jan.ai/api-reference)
-
-## Prerequisites
-
-### **Dependencies**
-
-Before installation, ensure that you have installed the following:
-- **Node.js**: version 18 and above is required to run the installation.
-- **NPM**: Needed to manage packages.
-- **CPU Instruction Sets**: Available for download from the [Cortex GitHub Releases](https://github.com/janhq/cortex/releases) page.
-
-
->💡 The **CPU instruction sets** are not required for the initial installation of Cortex. This dependency will be automatically installed during the Cortex initialization if they are not already on your system.
-
 
-### **Hardware**
-
-Ensure that your system meets the following requirements to run Cortex:
-
-- **OS**:
-  - MacOSX 13.6 or higher.
-  - Windows 10 or higher.
-  - Ubuntu 22.04 and later.
-- **RAM (CPU Mode):**
-  - 8GB for running up to 3B models.
-  - 16GB for running up to 7B models.
-  - 32GB for running up to 13B models.
-- **VRAM (GPU Mode):**
-
-  - 6GB can load the 3B model (int4) with `ngl` at 120 ~ full speed on CPU/ GPU.
-  - 8GB can load the 7B model (int4) with `ngl` at 120 ~ full speed on CPU/ GPU.
-  - 12GB can load the 13B model (int4) with `ngl` at 120 ~ full speed on CPU/ GPU.
-
-- **Disk**: At least 10GB for app and model download.
```

Comment:
I would love to maintain at least supported OS versions. This is a source of truth, even when the website is down somehow. As a oss user, I don't really like to get redirected to many endpoints.

Reply:
Yeah, sure, I think the prerequisites are important, will put it back in the docs.